.eslintrc

```diff
@@ -32,7 +32,19 @@
     // custom rules
     "@typescript-eslint/lines-between-class-members": "off",
     "@typescript-eslint/no-empty-function": "off",
+    "@typescript-eslint/no-throw-literal": "off",
     "@typescript-eslint/no-explicit-any": "error",
+    "@typescript-eslint/no-unused-vars": [
+      "error",
+      {
+        "args": "all",
+        "argsIgnorePattern": "^_",
+        "caughtErrors": "all",
+        "caughtErrorsIgnorePattern": "^_",
+        "destructuredArrayIgnorePattern": "^_",
+        "varsIgnorePattern": "^_"
```

Comment:
What does this incantation provide us?

Reply:
it allows us to ignore the variables that have `_` in front of it so they can be defined but not used. 
example:
```
try { } catch (error) { // we have to catch an error but silence it quietly }
```
after upgrading the version of the linter one it started to fail because we have some variables that were defined in catch/callbacks but were not used. this allows us to keep them there without failing the linter